source/installation/kafka-deployments/install-confluent-kafka.txt

```diff
@@ -15,3 +15,188 @@ Install with Confluent Cloud
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+You can install Relational Migrator with Kafka using 
+`Confluent Cloud <https://docs.confluent.io/cloud/current/get-started/confluent-cloud-basics.html#ccloud>`__. 
+This deployment method provides the reliability of Kafka without having to 
+manage your own Kafka cluster. 
+
+Before you Begin
+----------------
+
+Due to restrictions by Confluent, you are only able to use Basic, Standard, and 
+Dedicated clusters. For most use cases, the basic and standard cluster are enough 
+to handle migrating large volumes of data.
+
+Steps
+-----
+
+.. procedure::
+   :style: normal 
+
+   .. step:: Create a Confluent Cloud environment
+
+      Go to `Confluent Cloud <https://www.confluent.io/>`__ and create a new environment. 
+
+   .. step:: Enable schema registry
+
+      Enable schema registry and select a region.
+
+      If you have more than 1000 tables to migrate, use the Advanced schema registry tier.
+      Otherwise, the Essentials tier is enough. 
+
+   .. step:: Create a cluster
+
+      Navigate to your :guilabel:`Environment` page to click :guilabel:`Add Cluster`. 
+
+      For most use cases, the Basic and Standard tier clusters are enough. If you 
+      are migrating large volumes of data, consider the Dedicated cluster tier.
+
+   .. step:: Configure and gather permissions and API keys
+
+      To configure Relational Migrator, you must configure and gather the following 
+      permissions and API keys to include them in your ``user.properties`` file:
+
+      The following API keys are found in the Confluent Cloud console. For more 
+      information see, `Use API Keys <https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html>`__.
+
+      .. procedure::
+
+         .. step:: Configure Confluent Cloud API keys
+
+            Click the :icon-fa5:`bars` icon on the top right corner, then under 
+            :guilabel:`Administration`, select :guilabel:`Cloud API Keys`.
```

Comment:
**Praise**

Nice use of the icons here, this really helps users locate the correct spot in the UI!